BuildLoader.py

```diff
@@ -219,6 +219,7 @@ def __init__(self, *args, **kwargs):
         self.BPM_SIZE              = 0x1000 # valid only if ACM_SIZE > 0
         self.CFG_DATABASE_SIZE     = 0
 
+        self.FSP_M_STACK_TOP       = 0
         self.STAGE1A_XIP           = 1

```

Comment:
Had better update core DSC file to override PcdFSPMStackTop by this value.

Reply:
Good catch.  Added it in the new commit.